### ISSUE TITLE
DPS loading config.json multiple times to execute one single action

### ISSUE THREAD

**Issue by mageddo** [mageddo/dns-proxy-server#117]
For some reason DPS logs say it is loading config.json multiple times to solve a unique DNS record.

**What is expected**
DPS load config.json the mininum of times as possible

**What is happening**

It seems DPS is loading config.json multiple times

**Steps to reproduce**

```
$ nslookup google.com
```

log.txt
````
019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success
2019/04/04 06:13:11 INFO f=local.go:34 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration status=begin, confPath=/app/conf/config.json
2019/04/04 06:13:11 INFO f=local.go:37 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=openingFile
2019/04/04 06:13:11 INFO f=local.go:65 pkg=github.com/mageddo/dns-proxy-server/events/local m=LoadConfiguration  status=success```

**Comment by mageddo** [comment]
Dps is loading multiple times because it's really necessary, the point is the config.json have not a cache.

Develop a cache for the config file is necessary, this way we can reduce disk usage

**Comment by mageddo** [comment]
Fixed #267
